src/Plugin/CookiePlugin.php

```diff
@@ -69,7 +69,7 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
                     }
 
                     // Restrict setting cookie from another domain
-                    if (false === strpos($cookie->getDomain(), $request->getUri()->getHost())) {
+                    if (false === strpos($request->getUri()->getHost(), $cookie->getDomain())) {
```

Comment:
On the other hand, hack.symfony.slack.com should be correct again. Cookies.... 😕